src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-bucket.ts

```diff
@@ -35,3 +35,10 @@ export interface Bucket {
   read_tracker: number;
   bid: string;
 }
+
+export interface Notification {
+  Id: string;
+  Topic: string;
+  Event: string | string[];
+  Filter: string | string[];
```

Comment:
```suggestion
  Event: string[];
  Filter: Filter;
```
?? Filter is  a structured object  and why even is a string , either it should give array with mutiple, single, or none values.